app/javascript/miq-redux/reducer.ts

```diff
@@ -0,0 +1,25 @@
+import {createStore, Store, Reducer, Action} from 'redux';
+
+import { IMiqAction, AppReducer, IMiqReducerHash, AppState} from '../packs/miq-redux';
+
+const reducers: Set<AppReducer> = new Set();
+
+export const rootReducer: AppReducer = (state = {}, action: IMiqAction) => {
+    let newState = state;
+
+    reducers.forEach((reducer) => {
+        newState = reducer(newState, action)
```

Comment:
@ohadlevy thanks for the link. I'd prefer the API to be as simple as possible 😃 I guess we should give it some more thought.